fdbserver/KeyValueStoreRocksDB.actor.cpp

```diff
@@ -38,18 +39,21 @@ struct RocksDBKeyValueStore : IKeyValueStore {
 	using CF = rocksdb::ColumnFamilyHandle*;
 
 	struct Writer : IThreadPoolReceiver {
-		DB& db;
```

Comment:
This confused me because I read it as "reference to instance of rocksb", so I assumed the instance would be fully formed when this was constructed.  Instead, it's really a reference to a pointer to an instance, and we assign the pointer below.

Accessing `store->db` made it more obvious (to me) that we're directly manipulating store's pointer.